src/cards.ts

```diff
@@ -19,11 +21,33 @@ const sets = CardInfo.reduce(
     {} as Dictionary<Expansion>
 );
 
-const cards: Card[] = CardInfo.map(card => ({
+const validCards = CardInfo.filter(card => (
+    card.set !== "Dark Ages" && 
+    card.type === "kingdom" && 
+    !(card.set === "Promo" && card.name !== "Walled Village" && card.name !== "Stash") && 
+    !(card.set==="Intrigue 2nd Edition" && card.status === "removed")
```

Comment:
like @shocklateboy92  said all of these strings need to be constants, but I don't think it should be blocking as of now.